api/v1/standalone-dns-proxy/standalone-dns-proxy.proto

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+syntax = "proto3";
+
+package standalonednsproxy;
+
+option go_package = "github.com/cilium/cilium/api/v1/standalonednsproxy";
+
+service FQDNData {
+  rpc SubscribeToDNSPolicies(stream  DNSPoliciesResult) returns (stream DNSPolicies) {}
+  rpc UpdatesMappings(FQDNMapping) returns (UpdatesMappingsResult) {}
+}
+
+message DNSPoliciesResult {
+    bool success = 1;
+    string request_id = 2;
+}
+
+message FQDNMapping {
```

Comment:
In a sense, this tries to augment a DNS A/AAAA record with cilium specific information (i.e. source_identity, source_ip). Do we need to think about CNAMEs here, somehow?

Reply:
CNAME is used today only for hubble metrics report. I prefer to keep CNAME as out of scope for this PR.  can address in future as separate PR